README.md

```diff
@@ -16,6 +16,10 @@ consult the Krun documentation (fetched into `krun/` by `build.sh`), as there
 is a great deal of manual intervention needed to compile a tickless kernel,
 disable Intel P-states, set up `rc.local` etc.
 
+Note that the experiment is designed to run on amd64 machines running Debian 8
+or OpenBSD. Newer versions of Debian do not currently work due to a C++ ABI
+bump which would require us to use a newer C++ compiler.
```

Comment:
Can we say something like "which would require porting the experiment to use a newer C++ compiler" (I'm hoping if we don't say "us" that some kind soul might take it upon themselves to doing the porting!).

Reply:
Yeah, but I expect that if someone were to update it, they would not care for OpenBSD.

Updating GCC is easy if you don't care about OpenBSD.